### DISCUSSION TITLE
Dark Mode for On-device PDFs

### DISCUSSION THREAD

**Discussion by imadzanjani** [darkreader/darkreader#7371] [Q&A] [answered]
I know that we can use it for PDFs pulled form the web but it would be very nice to also be able to use with PDFs that are stored on the device and are being run using chrome. Is there any way to do that currently, if not can it be added in the next version?

**Comment by Gusted** [comment, marked as answer]
> I know that we can use it for PDFs pulled form the web but it would be very nice to also be able to use with PDFs that are stored on the device and are being run using chrome. Is there any way to do that currently, if not can it be added in the next version?

You've to enable local file access for the Dark Reader extension:

Open chrome://extensions page, find Dark Reader, click Allow access to file URLs.

**Reply by imadzanjani** [reply]
Oh thank you for that, I've been using PDFs in light mode for like a year now!
